spiders.html

```diff
@@ -100,9 +106,9 @@ <h1>All the Places Spiders</h1>
 
             knownBuilds.forEach(buildName => {
                 if (entry[1][buildName]) {
-                    row.push($("<td>").append(entry[1][buildName].features))
+                    row.push($("<td data-value=\"" + entry[1][buildName].features + "\">").append(entry[1][buildName].features))
                 } else {
-                    row.push($("<td>").append("&nbsp;"))
+                    row.push($("<td data-value=\"0\">").append("&nbsp;"))
```

Comment:
This was probably a mistake, no data for a spider because it's new, or has been deleted shouldn't be styled like a broken spider IMO.

Reply:
Ah yea, good point.